guides/common/modules/proc_configuring-repositories-proxy.adoc

```diff
@@ -0,0 +1,134 @@
+[id="configuring-repositories-proxy_{context}"]
+
+= Configuring Repositories
+
+ifeval::["{build}" == "foreman"]
+This procedure is only for Katello plug-in and {RHEL}-based operating system users.
+endif::[]
+
+Use this procedure to enable the repositories that are required to install {ProductName}.
+
+.Procedure
+To configure the required repositories, complete the following steps:
+
+. Disable all repositories:
++
+[options="nowrap"]
+----
+# subscription-manager repos --disable "*"
+----
++
+. Enable the following repositories:
+ifeval::["{build}" == "foreman"]
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# subscription-manager repos --enable={RepoRHEL7Server} \
+--enable rhel-7-server-optional-rpms \
+--enable rhel-7-server-extras-rpms
+----
++
+endif::[]
+ifeval::["{build}" == "satellite"]
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# subscription-manager repos --enable={RepoRHEL7Server} \
+--enable={RepoRHEL7ServerSatelliteCapsuleProductVersion} \
+--enable={RepoRHEL7ServerSatelliteMaintenanceProductVersion} \
+--enable={RepoRHEL7ServerSatelliteToolsProductVersion} \
+--enable={RepoRHEL7ServerSoftwareCollections} \
+--enable={RepoRHEL7ServerAnsible}
+----
++
+endif::[]
+
+NOTE: If you are installing {ProductName} as a virtual machine hosted on {oVirt}, you must also enable the *Red{nbsp}Hat Common* repository, and install {oVirt} guest agents and drivers. For more information, see https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.3/html/virtual_machine_management_guide/installing_guest_agents_and_drivers_linux#Installing_the_Guest_Agents_and_Drivers_on_Red_Hat_Enterprise_Linux[Installing the Guest Agents and Drivers on Red Hat Enterprise Linux] in the _Virtual Machine Management Guide_ for more information.
++
+. Clear any metadata:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-clean} all
+----
++
+. Optional: Verify that the required repositories are enabled:
++
+[options="nowrap"]
+----
+# yum repolist enabled
+----
+
+ifeval::["{build}" == "foreman"]
++
+. Install the `foreman-release.rpm` package:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# yum localinstall https://yum.theforeman.org/releases/{ProjectVersion}/el7/x86_64/foreman-release.rpm
+----
++
+. Install the `katello-repos-latest.rpm` package
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# yum localinstall https://fedorapeople.org/groups/katello/releases/yum/{KatelloVersion}/katello/el7/x86_64/katello-repos-latest.rpm
+----
++
+. Install the `puppet6-release-el-7.noarch.rpm` package:
++
+----
+# yum localinstall https://yum.puppet.com/puppet6-release-el-7.noarch.rpm
+----
++
+. Install the `epel-release-latest-7.noarch.rpm` package:
++
+----
+# yum localinstall https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+----
++
+. Install the `foreman-release-scl` package:
++
+----
+# yum install foreman-release-scl
```

Comment:
This should **not** be done on RHEL since this installs the CentOS SCL repositories instead of the supported RHEL ones.

Reply:
Should this step be removed or substituted by another command?

Reply:
You have `--enable={RepoRHEL7ServerSoftwareCollections} \` earlier so I think that should cover it but I must admit I've never installed Foreman on RHEL. I'm just going by https://theforeman.org/manuals/2.0/index.html#3.3.1RPMPackages

Reply:
I am fixing this in PR  #168.